src/cheri/app-cap-description.adoc

```diff
@@ -1,16 +1,21 @@
 [appendix]
-[#app_cap_description]
-== Anatomy of Capabilities in {cheri_base_ext_name}
+[#app_cap_description, reftext="Default capability encoding"]
+== Default Capability Encoding in {cheri_base_ext_name}
+
+This chapter describes the default capability encoding for both {cheri_base64_ext_name} and {cheri_base32_ext_name}.
+It covers the following configurations:
+
+* <<uycfg>>.base=0, and
+* <<uycfg>>.features=0 (no levels support) or 1 (<<section_zylevels1>> support)
```

Comment:
Those were, to be clear, strawperson proposals for encoding values.  It may be that we don't wish to offer the levels-less configurations?  (That can be a separate PR, tho'.)